resources/metadata/fortify.yaml

```diff
@@ -0,0 +1,442 @@
+      type: bool
+      description: Whether the classpath is automatically determined via build tool i.e. maven or pip or not at all
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: pythonRequirementsInstallSuffix
+      type: string
+      description: 'The suffix for the command used to install the requirements file in `scanType: ''pip''` to populate
+        the build environment with the necessary dependencies'
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: pythonVersion
+      type: string
+      description: 'Python version to be used in `scanType: ''pip''`'
+      scope:
+      - GENERAL
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: python3
+    - name: uploadResults
+      type: bool
+      description: Whether results shall be uploaded or not
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: buildDescriptorFile
+      type: string
+      description: 'Path to the build descriptor file addressing the module/folder
+        to be scanned. Defaults are for scanType=`maven`: `./pom.xml`, scanType=`pip`:
+        `./setup.py`, scanType=`mta`: determined automatically'
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+    - name: commitId
+      description: 'Set the Git commit ID for identifing artifacts throughout the scan.'
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: git/commitId
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      type: string
+    - name: commitMessage
+      description: 'Set the Git commit message for identifing pull request merges throughout the scan.'
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: git/commitMessage
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      type: string
+    - name: repoUrl
+      description: 'Set the source code repository URL for identifing sources of the scan.'
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: gitHttpsUrl
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      type: string
+    - name: repository
+      description: 'Set the GitHub repository for identifing artifacts throughout the scan.'
+      resourceRef:
+        - name: commonPipelineEnvironment
+          param: github/repository
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      type: string
+    - name: memory
+      type: string
+      description: The amount of memory granted to the translate/scan executions
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: -Xmx4G -Xms512M
+    - name: updateRulePack
+      type: bool
+      description: Whether the rule pack shall be updated and pulled from Fortify SSC before scanning or not
+      scope:
+      - PARAMETERS
+      - STAGES
+      - STEPS
+      default: true
+    - name: pythonExcludes
```

Comment:
should we rather name it `excludes` instead. Maybe this will get relevant for non-pip cases at some point?
For backward compatibility we could use an `alias`